.github/workflows/deploy.yml

```
@@ -0,0 +1,84 @@
+
+    # 6. ACR 로그인
+    - name: Login to Azure Container Registry
+      run: az acr login --name 23acr
+
+    # 7. Docker 이미지 빌드 및 푸시
+    - name: Build and push Docker image
+      run: |
+        # 고유한 태그 생성 (커밋 SHA + 타임스탬프)
+        IMAGE_TAG="23acr.azurecr.io/demo-ai-frontend:${{ github.sha }}"
+        LATEST_TAG="23acr.azurecr.io/demo-ai-frontend:latest"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)